src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class EmployeeService {
+    public static Employee getEmployeeByName(Employee[] employees, String name) {
```

Comment:
Don't use static methods in your solution
Static methods are in general a bad practice. Let's better create an instance of a class which method you want to call.